cmd/minishift/cmd/openshift/openshift.go

```diff
@@ -22,8 +22,8 @@ import (
 
 var OpenShiftCmd = &cobra.Command{
 	Use:   "openshift SUBCOMMAND [flags]",
-	Short: "Interact with an Openshift Cluster",
-	Long:  "Interact with an Openshift cluster, check available versions and modify master config resources",
+	Short: "Interacts with your local OpenShift cluster.",
```

Comment:
Technically the 'openshift' command does not do such thing. It is the container command for the actual subcommands which interact with OpenShift.